clients/drdisas/test_syntax.template

```diff
@@ -0,0 +1,13 @@
+#if defined(AARCH64)
+ f94017a0   ldr
```

Comment:
This doesn't look right: there should be operands.  Ditto with stp.  Is this really what DR outputs?

Reply:
Wow, I missed this. 

I tried running a sample with loglevel 3 to see how logs are being printed but hit an assert caused by dynamorio/core/arch/aarch64/decode.c:171

Command: `../../bin64/drrun -debug -loglevel 3 -syntax_arm -c libinscount.so -- ls`

I'll try to investigate further when I get a chance.

 BTW does -syntax_arm intended to be supported on AARCH64 please (couldn't find anything related on Github Issues) ?

Reply:
> BTW is -syntax_arm intended to be supported on AARCH64 please (couldn't find anything related on Github Issues) ?

Not sure what the status is there: @AssadHashmi ?

Reply:
Okay thanks! In the meantime, I am going to push the fixes for the other review comments.

Reply:
Added info to the issue page regarding this comment.